docs/standard/threading/using-threads-and-threading.md

```diff
@@ -9,32 +9,45 @@ ms.assetid: 9b5ec2cd-121b-4d49-b075-222cf26f2344
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
-# Using Threads and Threading
-The topics in this section discuss the creation and management of managed threads, how to pass data to managed threads and get results back, and how to destroy threads and handle a <xref:System.Threading.ThreadAbortException>.  
-  
-## In This Section  
- [Creating Threads and Passing Data at Start Time](../../../docs/standard/threading/creating-threads-and-passing-data-at-start-time.md)  
- Discusses and demonstrates the creation of managed threads, including how to pass data to new threads and how to get data back.  
-  
- [Pausing and Resuming Threads](../../../docs/standard/threading/pausing-and-resuming-threads.md)  
- Discusses the ramifications of pausing and resuming managed threads.  
-  
- [Destroying Threads](../../../docs/standard/threading/destroying-threads.md)  
- Discusses the ramifications of destroying managed threads, and how to handle a <xref:System.Threading.ThreadAbortException>.  
-  
- [Scheduling Threads](../../../docs/standard/threading/scheduling-threads.md)  
- Discusses thread priorities and how they affect thread scheduling.  
-  
-## Reference  
- <xref:System.Threading.Thread>  
- Provides reference documentation for the <xref:System.Threading.Thread> class, which represents a managed thread, whether it came from unmanaged code or was created in a managed application.  
-  
- <xref:System.Threading.ThreadStart>  
- Provides reference documentation for the <xref:System.Threading.ThreadStart> delegate that represents parameterless thread procedures.  
-  
- <xref:System.Threading.ParameterizedThreadStart>  
- Provides an easy way to pass data to a thread procedure, although without strong typing.  
-  
-## Related Sections  
- [Threads and Threading](../../../docs/standard/threading/threads-and-threading.md)  
- Provides an introduction to managed threading.
+# Using threads and threading
+
+With .NET you can write applications that perform multiple tasks at the same time. Tasks with the potential of holding up other tasks can execute on separate threads, a process known as *multithreading* or *free threading*.  
```

Comment:
nit: comma after .NET

I was thinking of tasks as Task and Task\<T> here until I realized this was written much earlier. I think that I'd use "operations" instead of "tasks" to avoid confusion.